incubator/exporter-otlp.spec/src/main/scripts/io/aklivity/zilla/specs/exporter/otlp/schema/otlp.schema.patch.json

```diff
@@ -0,0 +1,101 @@
+        "path": "/$defs/telemetry/exporter/allOf/-",
+        "value":
+        {
+            "if":
+            {
+               "properties":
+                {
+                    "type":
+                    {
+                        "const": "otlp"
+                    }
+                }
+            },
+            "then":
+            {
+                "properties":
+                {
+                    "type":
+                    {
+                        "const": "otlp"
+                    },
+                    "options":
+                    {
+                        "properties":
+                        {
+                            "interval":
+                            {
+                                "type": "number"
+                            },
+                            "signals":
+                            {
+                                "type": "array",
+                                "items":
+                                {
+                                    "enum":
+                                    [
+                                        "metrics",
+                                        "logs",
+                                        "traces"
+                                    ]
+                                }
+                            },
+                            "endpoint":
+                            {
+                                "type": "object",
+                                "properties":
+                                {
+                                    "protocol":
+                                    {
+                                        "enum":
+                                        [
+                                            "http",
+                                            "grpc"
+                                        ]
+                                    },
+                                    "location":
+                                    {
+                                        "type": "string"
+                                    },
+                                    "overrides":
+                                    {
+                                        "type": "object",
+                                        "properties":
+                                        {
+                                            "metrics":
+                                            {
+                                                "type": "string"
+                                            },
+                                            "logs":
+                                            {
+                                                "type": "string"
+                                            },
+                                            "traces":
+                                            {
+                                                "type": "string"
+                                            }
```

Comment:
We discussed this syntax as the endgame when we support `traces` and `logs`, but at the moment, the code insight we'd get from this schema would be confusing as we only support `metrics` at the moment.

Reply:
done, removed traces & logs from the schema as well as from the code